src/chart/generator/axis.cpp

```diff
@@ -88,7 +88,7 @@ bool DimensionAxis::add(const Data::MultiDim::SliceIndex &index,
 		it->second.weight = std::max(it->second.weight, enabled);
 		return false;
 	}
-	values.insert({index, Item(range, value, enabled)});
+	values.try_emplace(index, Item(range, value, enabled));
```

Comment:
Don't create here Item object. try _emplace will create it (so pass only the constructor arguments).